src/hacks.js

```diff
@@ -225,3 +226,7 @@ if (Button.isChild()) {
         window.Promise = ZalgoPromise;
     }
 }
+
+patchMethod(BillingPage, 'renderTo', ({ args: [ win, props, el ], original, context }) => {
```

Comment:
I think we can remove this. Just call `renderTo(window.top, ...)` when you render.